cclib/method/ddec.py

```diff
@@ -33,18 +33,28 @@ class DDEC6(Method):
     required_attrs = ("homos", "mocoeffs", "nbasis", "gbasis")
 
     def __init__(
-        self, data, volume, proatom_path=None, progress=None, loglevel=logging.INFO, logname="Log"
+        self,
+        data,
+        volume,
+        proatom_path=None,
+        progress=None,
+        convergence_level=1e-10,
+        loglevel=logging.INFO,
+        logname="Log",
     ):
-        # Inputs are:
-        # data -- ccData object that describe target molecule.
-        # volume -- Volume object that describe target Cartesian grid.
-        # proatom_path -- path to proatom densities
-        #      (directory containing atoms.h5 in horton or c2_001_001_000_400_075.txt in chargemol)
+        """ Initialize DDEC6 object.
+            Inputs are:
+                data -- ccData object that describe target molecule.
+                volume -- Volume object that describe target Cartesian grid.
+                proatom_path -- path to proatom densities
+                (directory containing atoms.h5 in horton or c2_001_001_000_400_075.txt in chargemol)
+        """
         super(DDEC6, self).__init__(data, progress, loglevel, logname)
 
         self.volume = volume
         self.fragresults = None
         self.proatom_path = proatom_path
+        self.convergence_level = convergence_level
 
         if numpy.sum(self.data.coreelectrons) != 0:
             # TODO: Pseudopotentials should be added back
```

Comment:
Could you create an issue for this please?

Reply:
I've added issue #925 regarding this. Thank you!